[CI Failure]: ARM CPU Test (test_moe.py)

### Name of failing test

`tests/kernels/moe/test_moe.py`

### Basic information

- [ ] Flaky test
- [ ] Can reproduce locally
- [ ] Caused by external libraries (e.g. bug in `transformers`)

### 🧪 Describe the failing test

This is appears to be a test infrastructure issue as the CI test fails with 
`ImportError: cannot import name 'TritonExperts' from 'vllm.model_executor.layers.fused_moe'` 


```
==================================== ERRORS ====================================
--
________________ ERROR collecting tests/kernels/moe/test_moe.py ________________
ImportError while importing test module '/vllm-workspace/tests/kernels/moe/test_moe.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/uv/python/cpython-3.12.12-linux-aarch64-gnu/lib/python3.12/importlib/__init__.py:90: in import_module
return _bootstrap._gcd_import(name[level:], package, level)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
tests/kernels/moe/test_moe.py:21: in <module>
from tests.kernels.moe.utils import (
tests/kernels/moe/utils.py:10: in <module>
from vllm.model_executor.layers.fused_moe import (
E   ImportError: cannot import name 'TritonExperts' from 'vllm.model_executor.layers.fused_moe' (/opt/venv/lib/python3.12/site-packages/vllm/model_executor/layers/fused_moe/__init__.py)
```

### 📝 History of failing test

The CI test first failed on Feb 12, 2026: https://buildkite.com/vllm/ci/builds/51276/summary?sid=019c50a7-77b7-41ff-9e84-7f9e2195b74c&tab=output

### CC List.

_No response_

--- a/tests/kernels/moe/utils.py
+++ b/tests/kernels/moe/utils.py
@@ -7,11 +7,6 @@ import vllm._custom_ops as ops
 from tests.kernels.quant_utils import per_block_cast_to_int8
 from tests.kernels.quantization.nvfp4_utils import FLOAT4_E2M1_MAX, FLOAT8_E4M3_MAX
 from vllm.model_executor.layers.activation import SiluAndMul
-from vllm.model_executor.layers.fused_moe import (
-    TritonExperts,
-    fused_experts,
-    fused_topk,
-)
 from vllm.model_executor.layers.fused_moe.activation import MoEActivation
 from vllm.model_executor.layers.fused_moe.config import (
     FusedMoEConfig,
@@ -24,10 +19,15 @@ from vllm.model_executor.layers.fused_moe.fused_batched_moe import (
     BatchedTritonExperts,
     NaiveBatchedExperts,
 )
+from vllm.model_executor.layers.fused_moe.fused_moe import (
+    TritonExperts,
+    fused_experts,
+)
 from vllm.model_executor.layers.fused_moe.modular_kernel import FusedMoEModularKernel
 from vllm.model_executor.layers.fused_moe.prepare_finalize import (
     MoEPrepareAndFinalizeNoEP,
 )
+from vllm.model_executor.layers.fused_moe.router.fused_topk_router import fused_topk
 from vllm.model_executor.layers.fused_moe.utils import moe_kernel_quantize_input
 from vllm.utils.deep_gemm import per_block_cast_to_fp8
 from vllm.utils.math_utils import round_up

